node_modules/col-canvas/lib/poller.js

```diff
@@ -626,106 +626,133 @@ var handleSubmission = function(ctx, users, assignment, activities, submissionAs
     'objectType': CollabosphereConstants.ACTIVITY.OBJECT_TYPES.CANVAS_SUBMISSION,
     'metadata': {
       'submission_id': submission.id,
-      'attempt': submission.attempt
+      'attempt': submission.attempt,
+      'sync_files': assignment.category.visible
```

Comment:
Note: I understand that 'sync_files' is meant as an action description. But lazy devs (like me) might not immediately recognize that intention.